_sass/base/template.scss

```diff
@@ -311,6 +311,18 @@ main
         li {
             margin: 5px 0;
         }
+        
+        thead {
+            text-align: left;
+        }
+        
+        td {
+            vertical-align: top;
+        }
+        
+        #ibm-manuals + table th:first-child  {
+            width : 8em;
```

Comment:
the first column is too small when viewed on a mac. Tested with safari and firefox